app/controllers/IndexController.scala

```diff
@@ -47,17 +48,25 @@ class IndexController @Inject() (
       val fatcaId = request.fatcaId
       subscriptionService.getSubscription(fatcaId).flatMap {
         sub =>
-          val changeContactDetailsUrl = if (sub.isBusiness) conf.changeOrganisationDetailsUrl else conf.changeIndividualDetailsUrl
-          val addNewFIUrl             = controllers.addFinancialInstitution.routes.AddFIController.onPageLoad.url
+          val changeContactDetailsUrl       = if (sub.isBusiness) conf.changeOrganisationDetailsUrl else conf.changeIndividualDetailsUrl
+          val addNewFIUrl                   = "/manage-your-crs-and-fatca-financial-institutions/add"
```

Comment:
Would this URL path be better configured in application.conf?

Reply:
Yes it would 👍